contributing.md

```diff
@@ -0,0 +1,79 @@
+
+
+## Adding reader tests
+
+Reader tests come in two flavors:
+
+ 1. Generated tables, which are specified from PySpark code in `dat/generated_tables.py`.
+ 2. External tables, which are checked in directly at `external-tables`.
```

Comment:
can we just remove all references to external tables, until we actually know what they are and what we expect to do to them?

Reply:
Yes, I had intended to but missed this :)